README.md

```diff
@@ -15,6 +15,49 @@ programatically see the [metadata-registry-tools] repository.
 
 New statements can be contributed by opening a pull request.
 
+To create an entity metadata statement using [metadata-registry-tools], use the
+following steps:
+
+1. Create an `entity-metadata.json` file describing the metadata statement, for
+   example something like (most fields can be omitted):
+
+   ```json
+   {
+     "v": 1,
+     "serial": 1,
+     "name": "My entity name",
+     "url": "https://my.entity/url",
+     "email": "my@entity.org",
+     "keybase": "myentity",
+     "twitter": "myentity"
+   }
+   ```
+
+   In case you are updating an existing entry, you must bump the serial number.
+
+2. Fork this repository.
+
+3. Use the `oasis-registry` command to generate a new signed statement. The
```

Comment:
Maybe link to the `metadata-registry-tools` repo here.  Maybe also short build instructions should be added to the `metadata-registry-tools` readme (although is mostly just having go installed + running `make build`)